yt_dlp/extractor/magellantv.py

```diff
@@ -46,7 +47,13 @@ def _real_extract(self, url):
         data = traverse_obj(self._search_nextjs_data(webpage, video_id), (
             'props', 'pageProps', 'reactContext',
             (('video', 'detail'), ('series', 'currentEpisode')), {dict}), get_all=False)
-        formats, subtitles = self._extract_m3u8_formats_and_subtitles(data['jwpVideoUrl'], video_id)
+        formats = []
+        subtitles = {}
+        for url in set(traverse_obj(data, ((('manifests', ..., 'hls'), 'jwp_video_url'), {url_or_none}))):
+            fmts, subs = self._extract_m3u8_formats_and_subtitles(
+                url, video_id, 'mp4', m3u8_id='hls', fatal=False)
+            formats.extend(fmts)
+            self._merge_subtitles(subs, target=subtitles)
```

Comment:
Maybe add some error handling (eg, 2nd test is geo-blocked in UK), s/t like this:
```suggestion
        context = traverse_obj(self._search_nextjs_data(webpage, video_id), (
            'props', 'pageProps', 'reactContext', {dict})
        data = traverse_obj(context, (
            (('video', 'detail'), ('series', 'currentEpisode')), {dict}, any))
        formats = []
        subtitles = {}
        for url in set(traverse_obj(data, ((('manifests', ..., 'hls'), 'jwp_video_url'), {url_or_none}))):
            fmts, subs = self._extract_m3u8_formats_and_subtitles(
                url, video_id, 'mp4', m3u8_id='hls', fatal=False)
            formats.extend(fmts)
            self._merge_subtitles(subs, target=subtitles)

        if not formats:
            error_msg = traverse_obj(context, ('errorDetailPage', 'errorMessage'))
            error_txt = error_msg or '**no reason given**'
            if "isn't available in your country" in error_msg:
                self.raise_geo_restricted(msg=error_txt)
            else:
                self.raise_no_formats(f'Extraction failed because: {error_txt}', expected=bool(error_msg))
```